People/README.md

```diff
@@ -68,6 +68,94 @@ on Android 11 or later.
 This sample uses the Gradle build system. To build this project, use the
 "gradlew build" command or use "Import Project" in Android Studio.
 
+## FAQ
+
+### Bubbles
+
+**Q. How can one determine within an activity, that it is in an expanded bubble?**
+
+    class BubbleActivity : AppCompatActivity() {
+      override fun onCreate(...) {
+      ...
+      val displayId = windowManager.defaultDisplay.displayId
+      val mIsBubbled = displayId != Display.DEFAULT_DISPLAY
```

Comment:
mIsBubbled → isBubbled
(Also in the inline below)

Reply:
done